galaxy_ng/app/common/pulp.py

```diff
@@ -15,9 +15,13 @@ def get_configuration():
     config.safe_chars_for_path_param = '/'
     if 'VERIFY_SSL' in settings:
         config.verify_ssl = settings.VERIFY_SSL
+    config.debug = True
```

Comment:
Since galaxy_pulp is removed, this code becomes invalid and doesn't seem to be needed.